src/SortableTrait.php

```diff
@@ -49,6 +50,32 @@ public function scopeOrdered(Builder $query, string $direction = 'asc')
     }
 
     /**
+     * Query records which are ordered above the given position.
+     *
+     * @param \Illuminate\Database\Eloquent\Builder $query
+     * @param int $position
+     *
+     * @return \Illuminate\Database\Query\Builder
+     */
+    public function scopeAbove(Builder $query, int $position)
```

Comment:
I don't like scopes being added that solely exist for internal working and will not be used publicly. Could you remove these?